### ISSUE TITLE
[bug]@api_chat.get("") do not  @requires(["authenticated"])

### ISSUE THREAD

**Issue by thinker007** [khoj-ai/khoj#890]
## Describe the bug
A clear and concise description of what the bug is. Please include what you were expecting to happen vs. what actually happened.
![image](https://github.com/user-attachments/assets/9b2e06e2-8210-4eb3-ba30-ad15e5a8e46d)

@api_chat.get("") 
do not requires(["authenticated"])

why and is this a bug?

**Comment by debanjum** [comment]
Thanks for point that out! The authenticated decorator got removed when we merged the websocket and the previous iteration of `api_chat.get()` in https://github.com/khoj-ai/khoj/pull/858. This would have thrown an internal server error as we expect user property to be always set to process chat requests. 

Saba just pushed a fix for it in 2d9dd81e768c5e4f3f70e1291a253c6461f40bde, so the error is more legible going forward and for better defense in depth
